website/content/en/status/report-2023-07-2023-09/mfsbsd-build-integration.adoc

```diff
@@ -0,0 +1,30 @@
+=== Integrate mfsBSD into the release building tools
```

Comment:
```suggestion
=== Integrate mfsBSD into the Release Building Tools
```

Changes of case to Americanise headings? 

For consistency within e.g. the table of contents of the compiled report. 

https://docs.freebsd.org/en/books/fdp-primer/book/#writing-style-guidelines